include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //uint32_t              maxInlineUniformTotalSize;
+    //VkBool32              integerDotProduct8BitUnsignedAccelerated;
+    //VkBool32              integerDotProduct8BitSignedAccelerated;
+    //VkBool32              integerDotProduct8BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProduct4x8BitPackedUnsignedAccelerated;
+    //VkBool32              integerDotProduct4x8BitPackedSignedAccelerated;
+    //VkBool32              integerDotProduct4x8BitPackedMixedSignednessAccelerated;
+    //VkBool32              integerDotProduct16BitUnsignedAccelerated;
+    //VkBool32              integerDotProduct16BitSignedAccelerated;
+    //VkBool32              integerDotProduct16BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProduct32BitUnsignedAccelerated;
+    //VkBool32              integerDotProduct32BitSignedAccelerated;
+    //VkBool32              integerDotProduct32BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProduct64BitUnsignedAccelerated;
+    //VkBool32              integerDotProduct64BitSignedAccelerated;
+    //VkBool32              integerDotProduct64BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating8BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating8BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating8BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating4x8BitPackedUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating4x8BitPackedSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating4x8BitPackedMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating16BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating16BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating16BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating32BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating32BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating32BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating64BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating64BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating64BitMixedSignednessAccelerated;
+    //VkDeviceSize          storageTexelBufferOffsetAlignmentBytes;
+    //VkBool32              storageTexelBufferOffsetSingleTexelAlignment;
+    //VkDeviceSize          uniformTexelBufferOffsetAlignmentBytes;
+    //VkBool32              uniformTexelBufferOffsetSingleTexelAlignment;
+    //VkDeviceSize          maxBufferSize;
```

Comment:
We want to expose this

Reply:
yes

Reply:
max of:
- [GL_MAX_SHADER_STORAGE_BLOCK_SIZE](https://opengl.gpuinfo.org/displaycapability.php?name=GL_MAX_SHADER_STORAGE_BLOCK_SIZE)
- [GL_MAX_UNIFORM_BLOCK_SIZE](https://opengl.gpuinfo.org/displaycapability.php?name=GL_MAX_UNIFORM_BLOCK_SIZE)
- [GL_MAX_TEXTURE_BUFFER_SIZE](https://opengl.gpuinfo.org/displaycapability.php?name=GL_MAX_TEXTURE_BUFFER_SIZE)